<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="entry">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="introduction">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Welcome To Yt-Downloader Made with PyQt6</string>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::MarkdownText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pasteButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>120</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Paste</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>120</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>421</width>
      <height>16</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="downloadButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>170</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QLabel" name="downloader">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>421</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Downloading</string>
    </property>
   </widget>
   <widget class="QLabel" name="support_links">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Supported Links - Youtube/Music/Playlists</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>694</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Window"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNew_Window">
   <property name="text">
    <string>New Window</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
